packages/web-react/src/components/Toast/constants.ts

```diff
@@ -12,3 +12,5 @@ export const ICON_BOX_SIZE = 20;
 export const TOAST_BAR_CLOSE_BUTTON_LABEL_DEFAULT = 'Close';
 
 export const DEFAULT_TOAST_COLOR = 'inverted';
+
+export const TOAST_AUTO_CLOSE_INTERVAL = 3000; // milliseconds
```

Comment:
```suggestion
export const DEFAULT_TOAST_AUTO_CLOSE_INTERVAL = 3000; // milliseconds
```

Could we please use the `DEFAULT_` prefix? I think it is more readable with this little change. Thanks.

Reply:
chenged